providers/atlassian/modules.go

```diff
@@ -4,21 +4,22 @@ import (
 	"github.com/amp-labs/connectors/common/paramsbuilder"
 )
 
-// ModuleJira is the module used for listing Jira issues.
-var ModuleJira = paramsbuilder.APIModule{ // nolint: gochecknoglobals
-	Label:   "rest/api",
-	Version: "3",
-}
-
-// ModuleEmpty is used for proxying requests through.
-var ModuleEmpty = paramsbuilder.APIModule{ // nolint: gochecknoglobals
-	Label:   "",
-	Version: "",
-}
+const (
+	// ModuleEmpty is used for proxying requests through.
+	ModuleEmpty paramsbuilder.APIModuleID = ""
+	// ModuleJira is the module used for listing Jira issues.
+	ModuleJira paramsbuilder.APIModuleID = "jira"
+)
 
 // supportedModules represents currently working and supported modules within the Atlassian connector.
 // Any added module should be appended here.
-var supportedModules = []paramsbuilder.APIModule{ // nolint: gochecknoglobals
-	ModuleEmpty,
-	ModuleJira,
+var supportedModules = paramsbuilder.APIModuleMapping{ // nolint: gochecknoglobals
```

Comment:
It'd be quite cool to use a `DefaultMap` such that we specify the fallback module. That way, we don't have to specify the fallback as an argument

Reply:
Nice, I will give it a thought

Reply:
So far right now I see that fallback will be used only when `WithModule` is called. But if it is not, then effectively the fallback doesn't kick in. I think I need to think better about it, and `DefaultMap` could come in handy.